tailwind.config.js

```diff
@@ -13,13 +13,26 @@ module.exports = {
       fontFamily:{
          sans:["Neufile Grotesk"] // called it sans for it to act as default
       },
+      letterSpacing:{
```

Comment:
remove the LetterSpacing configurations, because it is done in card #012. thus it will solve having conflicts in the config file.